include/RevProc.h

```diff
@@ -48,20 +48,22 @@
 #include "RevCoProc.h"
 #include "RevThread.h"
 #include "RevRand.h"
+#include "RevProcPasskey.h"
 #include "../common/syscalls/SysFlags.h"
 #include "../common/include/RevCommon.h"
 
 #define _PAN_FWARE_JUMP_            0x0000000000010000
 
 namespace SST::RevCPU{
+class RevCoProc;
```

Comment:
Is the forward decl necessary?

Reply:
Forward decl here is required for the RevProcPasskey<RevCoProc>